config/bucket_test.go

```diff
@@ -0,0 +1,69 @@
+package config_test
+
+import (
+	. "github.com/dogmatiq/dodeca/config"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("type Map", func() {
+	var bucket Map
+
+	BeforeEach(func() {
+		bucket = Map{
+			"<key-1>": String("<value-1>"),
+			"<key-2>": String("<value-2>"),
+			"<key-3>": Value{},
+		}
+	})
+
+	Describe("func Get()", func() {
+		It("returns the value associated with the key", func() {
+			v := bucket.Get("<key-1>")
+			Expect(v).To(Equal(String("<value-1>")))
+		})
+	})
+
+	Describe("func GetDefault()", func() {
```

Comment:
This doesn't specifically test a non defined key lookup to return a zero-value?
> // If they key is not defined, it returns a zero-value.

Reply:
It's implicit in the way a Go map works, but you're right - I will add that.

Reply:
See https://github.com/dogmatiq/dodeca/pull/37/files#diff-b8c85de65df4aab8016f61ebd47cbe89R26